### PR TITLE
Feature/futher ciper tests

### PR DESCRIPTION
**Pull request by RandomEDP**

### CODE REVIEW

File: features/steps/Ceaser_Cipher.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
+const { browser } = require('protractor');
+
 var chai = require('chai').use(require('chai-as-promised'));
 // import { Given, When, Then, BeforeAll, TableDefinition, AfterAll, } from "cucumber";
```
Comment (willfifield) [inline features/steps/Ceaser_Cipher.js]: Remove commented code

File: features/steps/Ceaser_Cipher.js [hunk 2]
```diff
@@ -1,3 +1,5 @@
+const { browser } = require('protractor');
```
Comment (willfifield) [inline features/steps/Ceaser_Cipher.js]: Use comment to describe what this testing file is for

File: features/pages/Ceaser_Cipher_Page.js [hunk 3]
```diff
@@ -1,4 +1,4 @@
-const { element } = require("protractor");
```
Comment (willfifield) [inline features/pages/Ceaser_Cipher_Page.js:1]: Add describing comment for file

**Review by willfifield — CHANGES_REQUESTED** [review]
Just a couple of minor things to align with coding standards

File: features/steps/Ceaser_Cipher.js (hunk 1 above)
Comment (RandomEDP) [reply features/steps/Ceaser_Cipher.js]: Resolved in new commit

File: features/steps/Ceaser_Cipher.js (hunk 2 above)
Comment (RandomEDP) [reply features/steps/Ceaser_Cipher.js]: Resolved in new commit

**Review by willfifield — APPROVED** [review]
Suggested changes have been updated.

**Pull request merged**